plugins/api-docs/src/components/ApisCards/presets.tsx

```diff
@@ -33,10 +33,12 @@ export function createSpecApiTypeColumn(): TableColumn<ApiEntity> {
 
 const ApiDefinitionButton = ({ apiEntity }: { apiEntity: ApiEntity }) => {
   const [dialogOpen, setDialogOpen] = useState(false);
-
   return (
     <>
-      <ToggleButton onClick={() => setDialogOpen(!dialogOpen)}>
+      <ToggleButton
+        aria-label="Toggle API Defination Dialog"
```

Comment:
```suggestion
        aria-label="Toggle API Definition Dialog"
```

Oh sorry I missed this typo first time I checked